src/tabs/AOCTab/SolutionView.jsx

```diff
@@ -1,45 +1,72 @@
-import Day1Solution from "./days/day1/Day1Script.js";
-import Day2Solution from "./days/day2/Day2Script.js";
-import Day3Solution from "./days/day3/Day3Script.js";
-import Day4Solution from "./days/day4/Day4Script.js";
+import React, { Suspense, useEffect, useState } from "react";
+import {ErrorBoundary} from 'react-error-boundary';
+import day1input from "./days/day1/input.txt";
+import day2input from "./days/day2/input.txt";
+import day3input from "./days/day3/input.txt";
+import day4input from "./days/day4/input.txt";
+import day1script from "./days/day1/day1script.txt";
+import day2script from "./days/day2/day2script.txt";
+import day3script from "./days/day3/day3script.txt";
+import day4script from "./days/day4/day4script.txt";
+import { CodeBlock, ocean } from "react-code-blocks";
 
-const solutions = [];
-solutions[1] = <Day1Solution />
-solutions[2] = <Day2Solution />
-solutions[3] = <Day3Solution />
-solutions[4] = <Day4Solution />
+const inputs = [

```

Comment:
This is definitely a fine workaround for the time being, let's put up a story on Trello to readdress it though